code/modules/mob/living/simple_animal/abnormality/aleph/mountain.dm

```diff
@@ -0,0 +1,259 @@
+	health = 800
+	pixel_x = -16
+	base_pixel_x = -16
+	damage_coeff = list(BRUTE = 1.2, RED_DAMAGE = 1.2, WHITE_DAMAGE = 0.8, BLACK_DAMAGE = 0.8, PALE_DAMAGE = 0.5)
+
+	melee_damage_lower = 24
+	melee_damage_upper = 24
+
+	melee_damage_type = RED_DAMAGE
+	armortype = RED_DAMAGE
+	stat_attack = DEAD
+	ranged = TRUE
+	attack_sound = 'sound/weapons/slashmiss.ogg'
+	attack_verb_continuous = "nips"
+	attack_verb_simple = "nips"
+	del_on_death = FALSE
+	can_breach = TRUE
+	threat_level = ALEPH_LEVEL
+	start_qliphoth = 2
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(0, 0, 0, 50, 55),
+						ABNORMALITY_WORK_INSIGHT = 0,
+						ABNORMALITY_WORK_ATTACHMENT = 0,
+						ABNORMALITY_WORK_REPRESSION = list(0, 0, 0, 50, 55)
+						)
+	work_damage_amount = 12
+	work_damage_type = BLACK_DAMAGE
+
+	ego_list = list(
+//		/datum/ego_datum/weapon/smile,
+//		/datum/ego_datum/armor/smile
+		)
+
+	var/finishing = FALSE
+	var/death_counter = 0
+	var/belly = 0
+	var/hurt = FALSE	//Are you hurt?
+	var/phase = 1
+	var/scream_cooldown
+	var/scream_cooldown_time = 10 SECONDS
+	var/scream_damage = 40
+
+/mob/living/simple_animal/hostile/abnormality/mountain/Initialize()		//1 in 100 chance for amogus MOSB
+	..()
+	if(prob(1))
+		icon_state = "amog"
+	return ..()
+
+
+/mob/living/simple_animal/hostile/abnormality/mountain/failure_effect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/mountain/work_complete(mob/living/carbon/human/user, work_type, pe, work_time)
+	if(user.stat == DEAD)
+		datum_reference.qliphoth_change(-1)
+
+	if(hurt == TRUE)
+		datum_reference.qliphoth_change(-1)
+		hurt = FALSE
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/mountain/attempt_work(mob/living/carbon/human/user, work_type)
+	if(user.health != user.maxHealth)
+		hurt = TRUE
+	return TRUE
+
+/mob/living/simple_animal/hostile/abnormality/mountain/CanAttack(atom/the_target)
+	if(finishing)
+		return FALSE
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/mountain/Move()
+	if(finishing)
+		return FALSE
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/mountain/breach_effect(mob/living/carbon/human/user)
+	..()
+	GiveTarget(user)
+	icon_state = "mosb_breach"
+
+/mob/living/simple_animal/hostile/abnormality/mountain/death(gibbed)
+	density = FALSE
+	animate(src, alpha = 0, time = 10 SECONDS)
+	QDEL_IN(src, 10 SECONDS)
+	..()
+
+//Nabbed from Big Bird
+/mob/living/simple_animal/hostile/abnormality/mountain/proc/on_mob_death(datum/source, mob/living/died, gibbed)
+	SIGNAL_HANDLER
+	if(!(status_flags & GODMODE)) // If it's breaching right now
+		return FALSE
+	if(!ishuman(died))
+		return FALSE
+	if(!died.ckey)
+		return FALSE
+	death_counter += 1
+	if(death_counter >= 10)
+		death_counter = 0
+		datum_reference.qliphoth_change(-1)
+	return TRUE
+
+
+/mob/living/simple_animal/hostile/abnormality/mountain/attackby(obj/item/I, mob/living/user, params)
```

Comment:
Not the proc you'd want to use, tbh.

Reply:
What would you prefer?
This is my first time running HP checks

Reply:
I don't want to run tick(), and the refactor that I did was buggy as hell, I think this is an okay proc to use